frontend/src/components/Home/ItemSearchCard.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import { CardHeader, Divider } from '@mui/material';
+import SearchBar from './SearchBar'
+
+class ItemSearchCard extends Component {
+    constructor(props) {
+        super(props);
+        //this.state = { inStock: false, value: "" };
+    }
+
+    render() {
+        const { classes } = this.props;
```

Comment:
IMO in situations like this, better to leave it out rather than having unnecessary unused boilerplate